fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/model/Sequence.java

```diff
@@ -32,11 +32,16 @@ public void apply(IDatabaseAdapter target) {
         target.createSequence(getSchemaName(), getObjectName(), this.cache);
     }
 
+    @Override
+    public void apply(Integer priorVersion, IDatabaseAdapter target) {
+        target.createSequence(getSchemaName(), getObjectName(), this.cache);
```

Comment:
I firmly believe we should have a NoOp here.  we should not create a sequence again.  only one time.